sp800-63d/vot_mapping.md

```diff
@@ -1,54 +1,82 @@
 
-## Assurance to VOT Mapping
+### Trustmark
 
-_This section is normative._
+The trustmark URI for these definitions is [[ URI for this document ]].
+
+### xAL to VoT Components
+
+This document uses three categories defined in VoT: Identity Proofing (P), Primary Credential Usage (C), and Assertion Presentation (A). These categories correspond to the Identity Assurance Level (IAL), Authenticator Assurance Level (AAL), and Federation Assurance Level (FAL), respectively. 
+
+This document does not use the Primary Credential Management (M) category defined in VoT.
+
+This document does not define any additional component categories. Any additional categories SHALL NOT be used.
 
-### xAL to VOT Components
+## Component Values
 
-|xAL|Component|
-|:--:|:----|
-|IAL|Identity Proofing|
-|AAL Authenticator Requirements|Primary Credential Usage|
-|AAL Authenticator Lifecycle|Primary Credential Management|
-|FAL|Assertion Presentation|
+This document does not use any of the existing values in VoT but instead defines new component values here in accordance with Section 8 of VoT.
 
 ### IAL to VOT Component Values
 
-May need to break some of these up by remote, remote supervised, in-person.
+IAL maps into the VoT component P using a numeric identifier. Only one numeric identifier SHALL be used. 
 
-IAL maps into the VOT component
+|IAL|Component Value|
+|:----:|:--:|
+|IAL1 (with no attributes)|P0|
+|IAL1|P1|
+|IAL2|P2|
+|IAL3|P3|
 
-|IAL|Options|Component Value|
-|:----:|:---|:--:|
-|IAL1|No attributes|P0|
-|IAL1|Self-asserted attributes|P1|
-|IAL2|Unsupervised Remote and In-Person|P2|
-|IAL3|Supervised Remote or In-person|P3|
+In addition, the type of proofing used MAY be indicated using an alphabetic identifier. Multiple alphabetic identifiers MAY be used simultaneously.
 
+|Proofing method|Component Value|
+|In-person|Pp|
+|Remote|Pr|
 
 ### AAL to VOT Component Values
 
-May need to break some of these up by optional requirements at lower levels
+AAL maps into the VoT component C using a numeric identifier. Only one numeric identifier SHALL be used. 
 
-|IAL|Vector|
+|AAL|Component Value|
 |:----:|:--:|
-|AAL1||
-|AAL2||
-|AAL3||
+|AAL1|C1|
+|AAL2|C2|
+|AAL3|C3|
+
+In addition, the type of authenticator used SHALL be indicated using an alphabetic identifier. Multiple aphabetic identifiers MAY be used simultaneously.
+
+|Authenticator|Component Value|
+|Memorized Secret|Cp|
+|Look-up Secret|Cu|
+|Out-of-band Device|Co|
+|Out-of-band over SMS|Cs|
+|Single-factor OTP|Ca|
+|Multi-factor OTP|Cb|
+|Single-factor Cryptographic Software|Cc|
+|Single-factor Cryptographic Device|Cd|
+|Multi-factor Cryptographic Software|Ce|
+|Multi-factor Cryptographic Device|Cf|
 
-**Need some language about secondary authenticators since VOT is focused on primary**
 
 ### FAL to VOT Component Values
 
-|IAL|Vector|
+FAL maps into the VoT component A using a numeric identifier. Only one numeric identifier SHALL be used. 
```

Comment:
Not a bad idea, though that's got more to do with the mechanics of presentation than the contents of the assertion. We could try it but my gut says I don't think we'd see much use or value in it. Still, add in now and edit out later.